SitumWayfinding/Classes/Positioning/IndicationsViewController.swift

```diff
@@ -0,0 +1,133 @@
+        self.indicationView.layer.borderWidth = 2
+        self.indicationView.layer.borderColor = UIColor.primaryDiminished.cgColor
+        self.indicationView.roundCorners(corners: [.topLeft, .topRight, .bottomLeft, .bottomRight])
+        self.indicationView.shadow()
+    
+        self.destinationLabel.font = .small
+        self.destinationLabel.textColor = .primary
+        self.indicationLabel.font = .bigBold
+        self.indicationLabel.textColor = .primary
+        self.indicationLabel.numberOfLines = 0
+        
+        self.indicationImage.tintColor = .primary
+        
+        self.indicationLoading.tintColor = .primary
+        if #available(iOS 13.0, *) {
+            self.indicationLoading.style = .large
+        }
+        self.indicationLoading.startAnimating()
+    }
+    
+    private func setupNextIndicationView() {
+        self.nextIndicationView.backgroundColor = .primary
+        self.nextIndicationView.roundCorners(corners: [.bottomLeft, .bottomRight])
+        self.nextIndicationView.shadow()
+        self.nextIndicationLabel.font = .small
+        self.nextIndicationLabel.textColor = .white
+    
+        self.nextIndicationImage.image = UIImage(named: "situm_direction_destination",
+            in: SitumMapsLibrary.bundle, compatibleWith: nil)
+        self.nextIndicationImage.tintColor = .white
+    }
+    
+    func setInstructions(progress: SITNavigationProgress, destination: String) {
+        if progress.currentIndication.action == .sitCalculating {
+            self.showLoading()
+        } else {
+            self.hideLoading()
+            self.setDestination(destination: destination)
+            self.indicationLabel.text = progress.currentIndication.humanReadableMessage()
+            self.indicationImage.image = self.getIndicationImage(indication: progress.currentIndication)
+            self.nextIndicationLabel.text = NSLocalizedString("navigation.nextInstruction",
+                bundle: SitumMapsLibrary.bundle,comment: "")
+            self.nextIndicationImage.image = self.getIndicationImage(indication: progress.nextIndication)
+        }
+    }
+    
+    func setDestination(destination: String) {
+        self.destinationLabel.text = destination
+    }
+    
+    private func showLoading() {
+        self.indicationLoading.isHidden = false
+        self.indicationImage.isHidden = true
+        self.indicationView.roundCorners(corners: [.topLeft, .topRight, .bottomLeft, .bottomRight])
+        self.indicationLabel.text = NSLocalizedString("navigation.loadingRoute",
+            bundle: SitumMapsLibrary.bundle, comment: "")
+        
+        self.nextIndicationView.isHidden = true
+    }
+    
+    private func hideLoading() {
+        self.indicationLoading.isHidden = true
+        self.indicationImage.isHidden = false
+        self.indicationView.roundCorners(corners: [.topLeft, .topRight, .bottomRight])
+        
+        self.nextIndicationView.isHidden = false
+    }
+    
```

Comment:
Sen ter exactamente na cabeza a estructura de clases do paquete, tería sentido crear un novo grupo/carpeta o mesmo nivel que Positioning chamado Navigation ou algo así?
Dividir esta clase en duas:
* controlador 
* noutra este último método de obtención de recursos, que parece que ten máis que ver co modelo

Valorao porque se cadra non ten moito sentido e é matar moscas a cañonazos

Reply:
Creo qué mellor falalo na revisión porque o que si que vexo e que esta clase podería ir en `SitumWayfinding/Classes/Positioning/Navigation/IndicationsViewController.swift` pero non teño moi claro como separar isto do modelo agora mesmo

Reply:
Falamolo na revision enton